examples/github-to-discord/packages/service/test/hello.test.ts

```diff
@@ -0,0 +1,32 @@
+import { TestEnvironment } from "@eventual/testing";
```

Comment:
Should we add some tests to the example? Would help to demonstrate?

Reply:
We don't have support for commands in tests right now and this example is only a single command. Lets add them when we support commands.